### PR TITLE
py-tensorflow-probability: new package

### PR DESCRIPTION
**Pull request by aweits**
+dependencies py-dm-tree, cloudpickle@1.1.1

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-dm-tree/package.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            """'--spawn_strategy=local',\n""",
+            # Ask bazel to explain what it's up to
+            # Needs a filename as argument
+            """'--explain=explainlogfile.txt',\n""",
+            # Increase verbosity of explanation,
+            """'--verbose_explanations',\n""",
+            # bazel uses system PYTHONPATH instead of spack paths
+            """'--action_env', 'PYTHONPATH={0}',\n""".format(env['PYTHONPATH']),
+        ]
+        filter_file("""'build',""",
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-dm-tree/package.py]: Why are you using triple quotes here?

File: var/spack/repos/builtin/packages/py-tensorflow-probability/package.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    url      = "https://github.com/tensorflow/probability/archive/v0.12.1.tar.gz"
+
+    maintainers = ['aweits']
+
+    version('0.12.1', sha256='1fe89e85fd053bf36e8645a5a1a53b729bc254cf1516bc224fcbd1e4ff50083a')
+    version('0.8.0', sha256='f6049549f6d6b82962523a6bf61c40c1d0c7ac685f209c0084a6da81dd78181d',
+            url='https://github.com/tensorflow/probability/archive/0.8.0.tar.gz')
+
+    extends('python')
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tensorflow-probability/package.py:27]: Missing build dep on setuptools

File: var/spack/repos/builtin/packages/py-dm-tree/package.py (hunk 1 above)
Comment (aweits) [reply var/spack/repos/builtin/packages/py-dm-tree/package.py]: hm, habit when I have to escape things. looks cleaner without them.

File: var/spack/repos/builtin/packages/py-tensorflow-probability/package.py (hunk 2 above)
Comment (aweits) [reply var/spack/repos/builtin/packages/py-tensorflow-probability/package.py:27]: thx

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**